### PR TITLE
Updated to JDK11 

### LINKED ISSUE #194: Raise language level to Java 11
Many features, especially for the Lists, Streams and Optionals are only available in Java > 8. I would like to raise the default language level to Java 11 then.

*build.gradle*
```groovy
sourceCompatibility = JavaVersion.VERSION_11
targetCompatibility = JavaVersion.VERSION_1_8
```

What you think?

### PR DESCRIPTION
**Pull request by martingrossmann**
# Description

Raised the language level to JDK11.

Fixes #194 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by martingrossmann** [comment]
This is strange: The most important change is the update of the language level in `build.gradle`, but Github says it is already done. But it is not!

**Comment by Zsar** [comment]
FWIW: This shows the expected diff:
```cmd
git fetch origin feature/testerra2-jdk11
git diff HEAD..FETCH_HEAD -- build.gradle
```
so, yes, must be an issue with GitHub.

... Here it is for ease of reference:
```git
diff --git a/build.gradle b/build.gradle
index 1f0bdbd9b..572abf844 100644
--- a/build.gradle
+++ b/build.gradle
@@ -35,8 +35,8 @@ allprojects {

     compileJava.options.encoding = 'UTF-8'
     compileTestJava.options.encoding = "UTF-8"
-    sourceCompatibility = JavaVersion.VERSION_1_8
-    targetCompatibility = JavaVersion.VERSION_1_8
+    sourceCompatibility = JavaVersion.VERSION_11
+    targetCompatibility = JavaVersion.VERSION_11

 }

@@ -56,7 +56,6 @@ subprojects {
     repositories {
         mavenLocal()
         mavenCentral()
-        jcenter()
     }

     jacoco {
```

**Comment by Zsar** [comment]
Created a [Github bug ticket](https://support.github.com/ticket/personal/0/1696125). Maybe we could wait a day or two with merging, so they have a better chance to dig through their backend. (_If_ they do in time, that is. Definitely not advocating for waiting for longer than end of the week.)

File: build.gradle [hunk 1]
```diff
@@ -56,7 +56,6 @@ subprojects {
     repositories {
         mavenLocal()
         mavenCentral()
-        jcenter()
```
Comment (Zsar) [inline build.gradle:59]: Huh. What was that used for previously?

**Review by Zsar — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (martingrossmann) [reply build.gradle:59]: JCenter from Bintray was also a huge Maven repo but it is not public anymore.

**Review by mreiche — APPROVED** [review]

**Pull request merged**